src/Resources/views/CRUD/base_list.html.twig

```diff
@@ -275,8 +275,8 @@ file that was distributed with this source code.
                             <div class="col-sm-9">
                                 {% set withAdvancedFilter = false %}
                                 {% for filter in admin.datagrid.filters %}
-                                    {% set filterActive = ((filter.isActive() and filter.options['show_filter'] is null) or (filter.options['show_filter'] is same as(true))) and not admin.isDefaultFilter(filter.formName) %}
-                                    {% set filterVisible = filter.options['show_filter'] is same as(true) or filter.options['show_filter'] is null %}
+                                    {% set filterActive = filter.isActive() or filter.options['show_filter'] is same as (true) %}
+                                    {% set filterVisible = filter.options['show_filter'] is not same as(false) %}
```

Comment:
I find weird the fact we need two different variable for a pretty similar thing.
```
sonata-filter="{{ filterVisible ? 'true' : 'false' }}" style="display: {% if filterActive %}block{% else %}none{% endif %}"
```

Do you understand the difference ? What's the purpose of `filterVisible` ?

Reply:
Filter can has default value, but hidden via `show_filter` option

Reply:
Im think is deprecate feature to, because getFilterParameters() merge default filter values with request

Reply:
Maybe some renaming could be great
I asked for renaming `filterActive` in `filterDisplayed`. 
```
{% set filterActive = filter.isActive() or filter.options['show_filter'] is same as (true) %}
```
Because when I see this code, I understood that we displayed the filter if it's active or if you explicitely asked to display it.

But then, I discovered a variable called `filterVisible`.

If I understand correctly
- filterActive should be renamed filterDisplayed
- filterVisible should be rename filterCanBeDisplayed

Reply:
In think we can remove filterVisible with sonata-filter attribute (also in js code)

Its need to provided values of hided filters via request. Im think that depreacted because final values merges from default values with request

Reply:
It's also needed to know if we should display or not the filters container, so I think we should keep it.

If you really sure about removing it it's maybe a better idea to do it in another PR, smaller PR lead to quicker/easier merge :)

Reply:
`filterCanBeDisplayed` - incorrect name. Filter will hide always.

Reply:
Always becouse we no have ways to show them. Checkbox doesnt rendered

Reply:
When `filter.options['show_filter']` is true, the filter will be always displayed.
When `filter.options['show_filter']` is null, the filter will be displayed if I check the box in the dropdown (thanks to the js) or if there is a value.
When `filter.options['show_filter']` is false, the filter will indeed never be displayed. So the filter can't be displayed.

That's why, to me
```
{% set filterCanBeDisplayed = filter.options['show_filter'] is not same as(false) %}
```
or in another way
```
{% set filterCantBeDisplayed = filter.options['show_filter'] is same as(false) %}
```